### PR TITLE
Improve scoping of netadp for rbx in 'vm create'

### PR DESCRIPTION
**Pull request by ad-m**
@fredericosilva , what about moving that logic to public?

### CODE REVIEW

**Comment by fredericosilva** [comment]
Is private networking out of RBX?

**Comment by ad-m** [comment]
In the case of RBX:
* private networks are not available,
* creation of a virtual machine with a selected public IP address (available) requires manual filling in the "service" field.
The second case can be handled on the public-API side.

File: bin/vm/create/index.js [hunk 1]
```diff
@@ -109,7 +108,12 @@ module.exports = resource => Cli.createCommand('create', {
                 netadp.network = args.network;
                 netadp.service = 'private';
             } else {
-                netadp.service = 'public';
+                const services = await args.helpers.api.get('/service/');
+                const vm_service = services.find(service => service.name === args.type || service._id === args.type);
```
Comment (ad-m) [inline bin/vm/create/index.js]: In my opinion it should go to public-API.

Comment (fredericosilva) [reply bin/vm/create/index.js]: IMO is already there ;)

**Comment by fredericosilva** [comment]
"cherry picked" on 682d211477ff8073d60d241495fb1212595ae9c7

**Pull request closed without merging**